src/Module/BaseAdminModule.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\Core\Addon;
+use Friendica\Core\L10n;
+use Friendica\Core\Renderer;
+use Friendica\Core\System;
+use Friendica\Database\DBA;
+
+abstract class BaseAdminModule extends BaseModule

```

Comment:
At least a PHP doc here I think.
=> Why do we add this abstract class and how is it used
=> For later admin pages :-)

Reply:
On it.